.github/dependabot.yml

```diff
@@ -1,6 +1,8 @@
 version: 2
 updates:
   - package-ecosystem: "github-actions"
+    cooldown:
+      default-days: 7
     directory: "/"
     schedule:
       interval: "weekly"
```

Comment:
Change this to monthly to reduce the update spam that comes from specific subversions